sys/include/net/gnrc/ppp/fsm.h

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos <jialamos@uc.cl>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_ppp Option Negotiation Automaton FSM header definitions
+ * @ingroup     net_gnrc_ppp
+ * @{
+ *
+ * @file
+ * @brief  Option Negotiation Automaton FSM definitions
+ *
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+#ifndef PPP_FSM_H_
+#define PPP_FSM_H_
+
+#include "net/gnrc/ppp/opt.h"
+#include "net/gnrc/ppp/prot.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/*Function flags*/
+#define F_TLU (1U << 0)
```

Comment:
Here and below: proper prefixing might help prevent future naming collisions. Especially for such shortly named macros.